visualLocTranslational.m

```diff
@@ -72,11 +75,13 @@
 
         fprintf('\n - Running Block %.0f \n', iBlock);
 
-        eyeTracker('StartRecording', cfg);
+        eyeTracker('Message', cfg, ['start_block-', iBlock]);
 
         % For each event in the block
         for iEvent = 1:cfg.design.nbEventsPerBlock
 
+            eyeTracker('Message', cfg, ['start_trial-', iEvent, '_', thisEvent]);
```

Comment:
I think you will have a problem there, no?

iEvent is an `integer` and `thisEvent` is a structure so concatenating those will throw an error, no?